components/script/dom/webglrenderingcontext.rs

```diff
@@ -2343,6 +2343,19 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             return NullValue();
         }
 
+        // special case that returns `WebGLRenderbuffer` or `WebGLTexture` dom object
+        match pname {
+            constants::FRAMEBUFFER_ATTACHMENT_OBJECT_NAME => {
+                if self.bound_renderbuffer.get().is_some() {
+                    return object_binding_to_js_or_null!(cx, &self.bound_renderbuffer);
+                } else {
+                    let texture = self.bound_texture(constants::TEXTURE_2D);
+                    return optional_root_object_to_js_or_null!(cx, texture);
```

Comment:
So, the spec says:

> If the value of FRAMEBUFFER_ATTACHMENT_OBJECT_TYPE is NONE, then querying any other pname will generate INVALID_ENUM.

Which means that if there's no texture and no renderbuffer, we should return and signal an invalid enum error for all the parameter names too.